godot/export_presets.cfg

```diff
@@ -410,6 +410,20 @@ permissions/write_user_dictionary=false
 xr_features/hand_tracking=0
 xr_features/hand_tracking_frequency=0
 xr_features/passthrough=0
+xr_features/enable_khronos_plugin=false
+xr_features/enable_lynx_plugin=false
+xr_features/enable_meta_plugin=false
+meta_xr_features/eye_tracking=0
+meta_xr_features/hand_tracking=0
+meta_xr_features/hand_tracking_frequency=0
+meta_xr_features/passthrough=0
+meta_xr_features/use_anchor_api=false
+meta_xr_features/use_scene_api=false
+meta_xr_features/quest_1_support=false
+meta_xr_features/quest_2_support=true
+meta_xr_features/quest_3_support=true
+meta_xr_features/quest_pro_support=true
```

Comment:
Should we disable the XR features for mobile export?

Reply:
Those flags will be ignored in Android because they are injected from the `addons/godotopenxrvendors/`, and that package is not included in non-XR builds (`godot/addons/dcl_build_plugin/build.gd` is the responsible for doing that)